modules/multi-architecture-modify-machine-set-aws.adoc

```diff
@@ -0,0 +1,120 @@
+[source,terminal]
+----
+$ oc create -f aws-arm64-machine-set-0.yaml
+----
+
++
+.Sample YAML compute machine set to deploy a ARM64 compute node
+
+[source,yaml]
+----
+apiVersion: machine.openshift.io/v1beta1
+kind: MachineSet
+metadata:
+  labels:
+    machine.openshift.io/cluster-api-cluster: <infrastructure_id> 
+  name: <infrastructure_id>-aws-arm64-machine-set-0 
+  namespace: openshift-machine-api
+spec:
+  replicas: 1
+  selector:
+    matchLabels:
+      machine.openshift.io/cluster-api-cluster: <infrastructure_id> 
+      machine.openshift.io/cluster-api-machineset: <infrastructure_id>-<role>-<zone> 
+  template:
+    metadata:
+      labels:
+        machine.openshift.io/cluster-api-cluster: <infrastructure_id> 
+        machine.openshift.io/cluster-api-machine-role: <role> 
+        machine.openshift.io/cluster-api-machine-type: <role> 
+        machine.openshift.io/cluster-api-machineset: <infrastructure_id>-<role>-<zone> 
+    spec:
+      metadata:
+        labels:
+          node-role.kubernetes.io/<role>: "" 
+      providerSpec:
+        value:
+          ami:
+            id: ami-02a574449d4f4d280 <1>
+          apiVersion: awsproviderconfig.openshift.io/v1beta1
+          blockDevices:
+            - ebs:
+                iops: 0
+                volumeSize: 120
+                volumeType: gp2
+          credentialsSecret:
+            name: aws-cloud-credentials
+          deviceIndex: 0
+          iamInstanceProfile:
+            id: <infrastructure_id>-worker-profile 
+          instanceType: m6g.xlarge <2> 
+          kind: AWSMachineProviderConfig
+          placement:
+            availabilityZone: us-east-1a <3>
+            region: <region> <4> 
+          securityGroups:
+            - filters:
+                - name: tag:Name
+                  values:
+                    - <infrastructure_id>-worker-sg 
+          subnet:
+            filters:
+              - name: tag:Name
+                values:
+                  - <infrastructure_id>-private-<zone> 
+          tags:
+            - name: kubernetes.io/cluster/<infrastructure_id> 
+              value: owned
+            - name: <custom_tag_name> 
+              value: <custom_tag_value> 
+          userDataSecret:
+            name: worker-user-data
+----
+
+
+<1> Specify an ARM64 supported Red Hat Enterprise Linux CoreOS (RHCOS) Amazon Machine Image (AMI) for your AWS zone for your OpenShift Container Platform nodes. 
++
+[source,terminal]
+----
+$ oc -n openshift-machine-config-operator get configmap/coreos-bootimages -o jsonpath='{.data.stream}' | jq -r '.architectures.<arch>.images.aws.regions."<region>".image'
+----
+<2> Specify an ARM64 supported machine type. For more information, refer to "Tested instance types for AWS 64-bit ARM"
+<3> Specify the zone, for example `us-east-1a`. Ensure that the zone you select offers 64-bit ARM machines. 
+<4> Specify the region, for example, `us-east-1`. Ensure that the zone you select offers 64-bit ARM machines.
```

Comment:
The callouts etc aren't tied to the step (see the indent in the preview). To get the codeblock and everything to behave, remove the extra spaces and surround this with `--` as follows:

```suggestion
+
--
.Sample YAML compute machine set to deploy a ARM64 compute node
[source,yaml]
----
apiVersion: machine.openshift.io/v1beta1
kind: MachineSet
metadata:
  labels:
    machine.openshift.io/cluster-api-cluster: <infrastructure_id> 
  name: <infrastructure_id>-aws-arm64-machine-set-0 
  namespace: openshift-machine-api
spec:
  replicas: 1
  selector:
    matchLabels:
      machine.openshift.io/cluster-api-cluster: <infrastructure_id> 
      machine.openshift.io/cluster-api-machineset: <infrastructure_id>-<role>-<zone> 
  template:
    metadata:
      labels:
        machine.openshift.io/cluster-api-cluster: <infrastructure_id> 
        machine.openshift.io/cluster-api-machine-role: <role> 
        machine.openshift.io/cluster-api-machine-type: <role> 
        machine.openshift.io/cluster-api-machineset: <infrastructure_id>-<role>-<zone> 
    spec:
      metadata:
        labels:
          node-role.kubernetes.io/<role>: "" 
      providerSpec:
        value:
          ami:
            id: ami-02a574449d4f4d280 <1>
          apiVersion: awsproviderconfig.openshift.io/v1beta1
          blockDevices:
            - ebs:
                iops: 0
                volumeSize: 120
                volumeType: gp2
          credentialsSecret:
            name: aws-cloud-credentials
          deviceIndex: 0
          iamInstanceProfile:
            id: <infrastructure_id>-worker-profile 
          instanceType: m6g.xlarge <2> 
          kind: AWSMachineProviderConfig
          placement:
            availabilityZone: us-east-1a <3>
            region: <region> <4> 
          securityGroups:
            - filters:
                - name: tag:Name
                  values:
                    - <infrastructure_id>-worker-sg 
          subnet:
            filters:
              - name: tag:Name
                values:
                  - <infrastructure_id>-private-<zone> 
          tags:
            - name: kubernetes.io/cluster/<infrastructure_id> 
              value: owned
            - name: <custom_tag_name> 
              value: <custom_tag_value> 
          userDataSecret:
            name: worker-user-data
----
<1> Specify an ARM64 supported Red Hat Enterprise Linux CoreOS (RHCOS) Amazon Machine Image (AMI) for your AWS zone for your OpenShift Container Platform nodes. 
+
[source,terminal]
----
$ oc -n openshift-machine-config-operator get configmap/coreos-bootimages -o jsonpath='{.data.stream}' | jq -r '.architectures.<arch>.images.aws.regions."<region>".image'
----
<2> Specify an ARM64 supported machine type. For more information, refer to "Tested instance types for AWS 64-bit ARM"
<3> Specify the zone, for example `us-east-1a`. Ensure that the zone you select offers 64-bit ARM machines. 
<4> Specify the region, for example, `us-east-1`. Ensure that the zone you select offers 64-bit ARM machines.
--
```